lib/diagnostic.rb

```diff
@@ -43,6 +43,10 @@ def transform(words)
   # replacing the trailing newline character with a
   # space and return the result.
   def read_file(filename)
-    # Your code goes here
+    File.open(filename) do |file|
+      file.each do |line, memo|
+          line.gsub!(/\n/, " ")
+          p line
+        end
+    end
```

Comment:
`File.open(filename)` gives us the single file that we're loading. The File class is Enumerable, so we can call Enumerable methods on files.